_include/q/gradle-quickstart.sh

```diff
@@ -38,14 +38,22 @@ function mkPackage() {
   echo ${x//./\/}
 }
 
+function extract_parameter() {
+  if [[ "$#" != 1 ]]; then
+    echo "Fatal error: No parameter or too many parameter passed: $@"
+    exit 1;
+  fi
+  awk -F'"' '/'$1' = "/{print $2}' docs/config.toml
```

Comment:
```suggestion
  awk -F'"' '/'$1' = "/{print $2}' docs/config.toml
```
I didn't mention that in my initial proposal (sorry for that) but it might make sense to also validate the returned value (to ensure that we actually found a value).